<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleScrollValue</class>
 <widget class="QFrame" name="ModuleScrollValue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="labelValue1">
     <property name="font">
      <font>
       <family>MiSans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#B6B6B6</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelValue2">
     <property name="font">
      <font>
       <family>MiSans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#A6A6A6</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelLine1">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>3</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(
    spread: pad,
    x1: 0, y1: 0,  /* 渐变起点：左侧 */
    x2: 1, y2: 0,  /* 渐变终点：右侧 */
    stop: 0   rgba(246, 246, 246, 255),    /* 起点颜色：黑色 */
    stop: 0.5 rgba(44, 44, 44, 255),  /* 中间颜色：白色 */
    stop: 1   rgba(246, 246, 246, 255)     /* 终点颜色：黑色 */
);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelValue3">
     <property name="font">
      <font>
       <family>MiSans</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelLine2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>3</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(
    spread: pad,
    x1: 0, y1: 0,  /* 渐变起点：左侧 */
    x2: 1, y2: 0,  /* 渐变终点：右侧 */
    stop: 0   rgba(246, 246, 246, 255),    /* 起点颜色：黑色 */
    stop: 0.5 rgba(44, 44, 44, 255),  /* 中间颜色：白色 */
    stop: 1   rgba(246, 246, 246, 255)     /* 终点颜色：黑色 */
);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelValue4">
     <property name="font">
      <font>
       <family>MiSans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#A6A6A6</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelValue5">
     <property name="font">
      <font>
       <family>MiSans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#B6B6B6</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
